run_tests.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Check if in the right working directory.
+if [ ! -d "webapp" ]; then
+  echo "You are supposed to run this script in the Deadline root directory."
+  exit 1
+fi
+
+# Check if in Deadline virtual environment.
+source venv/deadline/bin/activate || exit 1;
```

Comment:
should this be venv/sloth to match the readme?

Reply:
great catch! I forgot to change that when I copy pasted it from my other project. It should actually be "Deadline", matching our project name.